### DISCUSSION TITLE
GRN velocity

### DISCUSSION THREAD

**Discussion by gilgolan73** [aertslab/scenicplus#516] [Q&A]
Hello, 
I saw on the SCENIC+ article (SCENIC+: single-cell multiomic inference of enhancers and gene regulatory networks) an implementation of "GRN velocity" for calculation of TF strength in differentiation.
Is there a tutorial? or an example code for using this method?

Thank you,
Gil

**Comment by fstrijth** [comment]
You can find the code for the analyses in the paper here https://github.com/aertslab/scenicplus_analyses/tree/main/EAD
